{"tests": [{"data": {"status": "failure", "inputs": {"input|zip_trinity": {"src": "hda", "id": "2891970512fa2d5a"}, "percent_identity": "100", "input|input_type_choice": "trinity", "length_seq_max": "100", "overlap_length": "60"}, "output_problems": ["Job in error state.", "Job in error state."], "job": {"inputs": {"zip_trinity": {"src": "hda", "id": "2891970512fa2d5a", "uuid": "9ae68fa2-39ad-46e2-a5f8-40f3ddc80c3a"}}, "update_time": "2016-12-06T13:36:46.158720", "tool_id": "filter_assemblies", "outputs": {"output": {"src": "hda", "id": "5729865256bc2525", "uuid": "5aa84e88-c392-4389-a696-ada1c3b94314"}}, "stdout": "done\n", "command_line": "python /home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/script_to_choose.py  trinity /tmp/tmpHRkY17/files/000/dataset_1.dat 100 100 60 > /tmp/tmpHRkY17/files/000/dataset_2.dat && echo \"done\"", "exit_code": 0, "state": "error", "create_time": "2016-12-06T13:36:41.466250", "params": {"percent_identity": "\"100\"", "overlap_length": "\"60\"", "dbkey": "\"hg17\"", "input": "{\"input_type_choice\": \"trinity\", \"zip_trinity\": {\"values\": [{\"src\": \"hda\", \"id\": \"2891970512fa2d5a\"}]}, \"__current_case__\": 1}", "chromInfo": "\"/tmp/tmpHRkY17/galaxy-dev/tool-data/shared/ucsc/chrom/hg17.len\"", "length_seq_max": "\"100\""}, "stderr": "python: can't open file '/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/01_Choose_One_variants_per_locus_TRINITY_v1.0.py': [Errno 2] No such file or directory\nTraceback (most recent call last):\n  File \"/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/find_orf.py\", line 45, in <module>\n    f_in = open(path_IN, \"r\")\nIOError: [Errno 2] No such file or directory: '04Pfiji_trinity.fasta'\nsh: 1: cap3: not found\nsh: 1: cannot open 05Pfiji_trinity.fasta.cap.singlets: No such file\npython: can't open file '/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/01_Choose_One_variants_per_locus_TRINITY_v1.0.py': [Errno 2] No such file or directory\nTraceback (most recent call last):\n  File \"/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/find_orf.py\", line 45, in <module>\n    f_in = open(path_IN, \"r\")\nIOError: [Errno 2] No such file or directory: '04Apomp_trinity.fasta'\nsh: 1: cap3: not found\nsh: 1: cannot open 05Apomp_trinity.fasta.cap.singlets: No such file\npython: can't open file '/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/01_Choose_One_variants_per_locus_TRINITY_v1.0.py': [Errno 2] No such file or directory\nTraceback (most recent call last):\n  File \"/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/find_orf.py\", line 45, in <module>\n    f_in = open(path_IN, \"r\")\nIOError: [Errno 2] No such file or directory: '04Amphi_trinity.fasta'\nsh: 1: cap3: not found\nsh: 1: cannot open 05Amphi_trinity.fasta.cap.singlets: No such file\npython: can't open file '/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/01_Choose_One_variants_per_locus_TRINITY_v1.0.py': [Errno 2] No such file or directory\nTraceback (most recent call last):\n  File \"/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/find_orf.py\", line 45, in <module>\n    f_in = open(path_IN, \"r\")\nIOError: [Errno 2] No such file or directory: '04Acaud_trinity.fasta'\nsh: 1: cap3: not found\nsh: 1: cannot open 05Acaud_trinity.fasta.cap.singlets: No such file\n", "job_metrics": [], "model_class": "Job", "external_id": "9063", "id": "5729865256bc2525", "user_email": "test@bx.psu.edu"}, "problem_log": "  File \"/usr/lib/python2.7/unittest/case.py\", line 327, in run\n    testMethod()\n  File \"/tmp/tmpHRkY17/galaxy-dev/test/functional/test_toolbox.py\", line 299, in test_tool\n    self.do_it( td )\n  File \"/tmp/tmpHRkY17/galaxy-dev/test/functional/test_toolbox.py\", line 75, in do_it\n    raise e\n'Job in error state.\\nJob in error state.\\n-------------------- >> begin captured stdout << ---------------------\\nHistory with id 2891970512fa2d5a in error - summary of datasets in error below.\\n--------------------------------------\\n| 2 - Filter_oase_zip (HID - NAME) \\n| Dataset Blurb:\\n|  error\\n| Dataset Info:\\n|  done\\n|  python: can\\'t open file \\'/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/01_Choose_One_variants_per_locus_TRINITY_v1.0.py\\': [Errno 2] No such file or directory\\n|  Traceback (most recent call l\\n| Dataset Job Standard Output:\\n|  done\\n| Dataset Job Standard Error:\\n|  python: can\\'t open file \\'/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/01_Choose_One_variants_per_locus_TRINITY_v1.0.py\\': [Errno 2] No such file or directory\\n|  Traceback (most recent call last):\\n|  File \"/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/find_orf.py\", line 45, in <module>\\n|  f_in = open(path_IN, \"r\")\\n|  IOError: [Errno 2] No such file or directory: \\'04Pfiji_trinity.fasta\\'\\n|  sh: 1: cap3: not found\\n|  sh: 1: cannot open 05Pfiji_trinity.fasta.cap.singlets: No such file\\n|  python: can\\'t open file \\'/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/01_Choose_One_variants_per_locus_TRINITY_v1.0.py\\': [Errno 2] No such file or directory\\n|  Traceback (most recent call last):\\n|  File \"/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/find_orf.py\", line 45, in <module>\\n|  f_in = open(path_IN, \"r\")\\n|  IOError: [Errno 2] No such file or directory: \\'04Apomp_trinity.fasta\\'\\n|  sh: 1: cap3: not found\\n|  sh: 1: cannot open 05Apomp_trinity.fasta.cap.singlets: No such file\\n|  python: can\\'t open file \\'/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/01_Choose_One_variants_per_locus_TRINITY_v1.0.py\\': [Errno 2] No such file or directory\\n|  Traceback (most recent call last):\\n|  File \"/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/find_orf.py\", line 45, in <module>\\n|  f_in = open(path_IN, \"r\")\\n|  IOError: [Errno 2] No such file or directory: \\'04Amphi_trinity.fasta\\'\\n|  sh: 1: cap3: not found\\n|  sh: 1: cannot open 05Amphi_trinity.fasta.cap.singlets: No such file\\n|  python: can\\'t open file \\'/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/01_Choose_One_variants_per_locus_TRINITY_v1.0.py\\': [Errno 2] No such file or directory\\n|  Traceback (most recent call last):\\n|  File \"/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/find_orf.py\", line 45, in <module>\\n|  f_in = open(path_IN, \"r\")\\n|  IOError: [Errno 2] No such file or directory: \\'04Acaud_trinity.fasta\\'\\n|  sh: 1: cap3: not found\\n|  sh: 1: cannot open 05Acaud_trinity.fasta.cap.singlets: No such file\\n|\\n--------------------------------------\\nHistory with id 2891970512fa2d5a in error - summary of datasets in error below.\\n--------------------------------------\\n| 2 - Filter_oase_zip (HID - NAME) \\n| Dataset Blurb:\\n|  error\\n| Dataset Info:\\n|  done\\n|  python: can\\'t open file \\'/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/01_Choose_One_variants_per_locus_TRINITY_v1.0.py\\': [Errno 2] No such file or directory\\n|  Traceback (most recent call l\\n| Dataset Job Standard Output:\\n|  done\\n| Dataset Job Standard Error:\\n|  python: can\\'t open file \\'/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/01_Choose_One_variants_per_locus_TRINITY_v1.0.py\\': [Errno 2] No such file or directory\\n|  Traceback (most recent call last):\\n|  File \"/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/find_orf.py\", line 45, in <module>\\n|  f_in = open(path_IN, \"r\")\\n|  IOError: [Errno 2] No such file or directory: \\'04Pfiji_trinity.fasta\\'\\n|  sh: 1: cap3: not found\\n|  sh: 1: cannot open 05Pfiji_trinity.fasta.cap.singlets: No such file\\n|  python: can\\'t open file \\'/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/01_Choose_One_variants_per_locus_TRINITY_v1.0.py\\': [Errno 2] No such file or directory\\n|  Traceback (most recent call last):\\n|  File \"/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/find_orf.py\", line 45, in <module>\\n|  f_in = open(path_IN, \"r\")\\n|  IOError: [Errno 2] No such file or directory: \\'04Apomp_trinity.fasta\\'\\n|  sh: 1: cap3: not found\\n|  sh: 1: cannot open 05Apomp_trinity.fasta.cap.singlets: No such file\\n|  python: can\\'t open file \\'/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/01_Choose_One_variants_per_locus_TRINITY_v1.0.py\\': [Errno 2] No such file or directory\\n|  Traceback (most recent call last):\\n|  File \"/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/find_orf.py\", line 45, in <module>\\n|  f_in = open(path_IN, \"r\")\\n|  IOError: [Errno 2] No such file or directory: \\'04Amphi_trinity.fasta\\'\\n|  sh: 1: cap3: not found\\n|  sh: 1: cannot open 05Amphi_trinity.fasta.cap.singlets: No such file\\n|  python: can\\'t open file \\'/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/01_Choose_One_variants_per_locus_TRINITY_v1.0.py\\': [Errno 2] No such file or directory\\n|  Traceback (most recent call last):\\n|  File \"/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/find_orf.py\", line 45, in <module>\\n|  f_in = open(path_IN, \"r\")\\n|  IOError: [Errno 2] No such file or directory: \\'04Acaud_trinity.fasta\\'\\n|  sh: 1: cap3: not found\\n|  sh: 1: cannot open 05Acaud_trinity.fasta.cap.singlets: No such file\\n|\\n--------------------------------------\\n\\n--------------------- >> end captured stdout << ----------------------\\n-------------------- >> begin captured logging << --------------------\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\ngalaxy.web.framework.webapp: INFO: Session authenticated using Galaxy master api key\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/users?key=test_key HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\ngalaxy.web.framework.webapp: INFO: Session authenticated using Galaxy master api key\\nrequests.packages.urllib3.connectionpool: DEBUG: \"POST /api/users HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\ngalaxy.web.framework.webapp: INFO: Session authenticated using Galaxy master api key\\nrequests.packages.urllib3.connectionpool: DEBUG: \"POST /api/users/5729865256bc2525/api_key HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"POST /api/histories HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\ngalaxy.tools: DEBUG: Validated and populated state for tool request (35.547 ms)\\ngalaxy.tools.actions.upload: DEBUG: Persisted uploads (15.317 ms)\\ngalaxy.tools.actions.upload: DEBUG: Checked and cleaned uploads (578.810 ms)\\ngalaxy.tools.actions.upload_common: INFO: tool upload1 created job id 1\\ngalaxy.tools.actions.upload: DEBUG: Created upload job (293.460 ms)\\ngalaxy.tools.execute: DEBUG: Tool [upload1] created job [1] (888.376 ms)\\ngalaxy.tools.execute: DEBUG: Executed 1 job(s) for tool upload1 request: (917.836 ms)\\nrequests.packages.urllib3.connectionpool: DEBUG: \"POST /api/tools HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\ngalaxy.jobs: DEBUG: (1) Working directory for job is: /tmp/tmpHRkY17/job_working_directory/000/1\\ngalaxy.jobs.handler: DEBUG: (1) Dispatching to planemo_runner runner\\ngalaxy.jobs: DEBUG: (1) Persisting job destination (destination id: upload_dest)\\ngalaxy.jobs.runners: DEBUG: Job [1] queued (170.791 ms)\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\ngalaxy.jobs.handler: INFO: (1) Job dispatched\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\ngalaxy.tools.deps: DEBUG: Building dependency shell command for dependency \\'samtools\\'\\ngalaxy.tools.deps: DEBUG: Find dependency samtools version None\\ngalaxy.tools.deps: DEBUG: Dependency samtools not found.\\ngalaxy.tools.deps: WARNING: Failed to resolve dependency on \\'samtools\\', ignoring\\ngalaxy.jobs.command_factory: INFO: Built script [/tmp/tmpHRkY17/job_working_directory/000/1/tool_script.sh] for tool command[python /tmp/tmpHRkY17/galaxy-dev/tools/data_source/upload.py /tmp/tmpHRkY17/galaxy-dev /tmp/tmpHRkY17/tmp/tmp95Y2R5 /tmp/tmpHRkY17/tmp/tmpe9kRwF 1:/tmp/tmpHRkY17/job_working_directory/000/1/dataset_1_files:/tmp/tmpHRkY17/files/000/dataset_1.dat]\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\ngalaxy.tools.deps: DEBUG: Building dependency shell command for dependency \\'samtools\\'\\ngalaxy.tools.deps: DEBUG: Find dependency samtools version None\\ngalaxy.tools.deps: DEBUG: Dependency samtools not found.\\ngalaxy.tools.deps: WARNING: Failed to resolve dependency on \\'samtools\\', ignoring\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\ngalaxy.jobs.runners: DEBUG: (1) command is: mkdir -p working; cd working; /tmp/tmpHRkY17/job_working_directory/000/1/tool_script.sh; return_code=$?; cd \\'/tmp/tmpHRkY17/job_working_directory/000/1\\'; python \"/tmp/tmpHRkY17/job_working_directory/000/1/set_metadata_hJgu30.py\" \"/tmp/tmpHRkY17/tmp/tmp95Y2R5\" \"/tmp/tmpHRkY17/job_working_directory/000/1/working/galaxy.json\" \"/tmp/tmpHRkY17/job_working_directory/000/1/metadata_in_HistoryDatasetAssociation_1_JI5vk8,/tmp/tmpHRkY17/job_working_directory/000/1/metadata_kwds_HistoryDatasetAssociation_1_ri66IF,/tmp/tmpHRkY17/job_working_directory/000/1/metadata_out_HistoryDatasetAssociation_1_FTH5__,/tmp/tmpHRkY17/job_working_directory/000/1/metadata_results_HistoryDatasetAssociation_1_KiHeOR,/tmp/tmpHRkY17/files/000/dataset_1.dat,/tmp/tmpHRkY17/job_working_directory/000/1/metadata_override_HistoryDatasetAssociation_1_IjZ9rj\" 5242880; sh -c \"exit $return_code\"\\ngalaxy.jobs.runners.local: DEBUG: (1) executing job script: /tmp/tmpHRkY17/job_working_directory/000/1/galaxy_1.sh\\ngalaxy.jobs: DEBUG: (1) Persisting job destination (destination id: upload_dest)\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\ngalaxy.jobs.runners.local: DEBUG: execution finished: /tmp/tmpHRkY17/job_working_directory/000/1/galaxy_1.sh\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\ngalaxy.model.metadata: DEBUG: loading metadata from file for: HistoryDatasetAssociation 1\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\ngalaxy.jobs: DEBUG: job 1 ended (finish() executed in (664.497 ms))\\ngalaxy.tools: DEBUG: Validated and populated state for tool request (145.637 ms)\\ngalaxy.tools.actions: INFO: Handled output named output for tool filter_assemblies (123.394 ms)\\ngalaxy.tools.actions: INFO: Added output datasets to history (96.733 ms)\\ngalaxy.tools.actions: INFO: Verified access to datasets for Job[unflushed,tool_id=filter_assemblies] (9.208 ms)\\ngalaxy.tools.actions: INFO: Setup for job Job[unflushed,tool_id=filter_assemblies] complete, ready to flush (29.209 ms)\\ngalaxy.tools.actions: INFO: Flushed transaction for job Job[id=2,tool_id=filter_assemblies] (116.729 ms)\\ngalaxy.tools.execute: DEBUG: Tool [filter_assemblies] created job [2] (372.282 ms)\\ngalaxy.tools.execute: DEBUG: Executed 1 job(s) for tool filter_assemblies request: (403.545 ms)\\nrequests.packages.urllib3.connectionpool: DEBUG: \"POST /api/tools HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/jobs/5729865256bc2525?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\ngalaxy.jobs: DEBUG: (2) Working directory for job is: /tmp/tmpHRkY17/job_working_directory/000/2\\ngalaxy.jobs.handler: DEBUG: (2) Dispatching to planemo_runner runner\\ngalaxy.jobs: DEBUG: (2) Persisting job destination (destination id: planemo_dest)\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\ngalaxy.jobs.runners: DEBUG: Job [2] queued (146.811 ms)\\ngalaxy.jobs.handler: INFO: (2) Job dispatched\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/jobs/5729865256bc2525?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\ngalaxy.tools.deps: DEBUG: Building dependency shell command for dependency \\'python\\'\\ngalaxy.tools.deps: DEBUG: Find dependency python version 2.7\\ngalaxy.tools.deps: DEBUG: Dependency python not found.\\ngalaxy.tools.deps: WARNING: Failed to resolve dependency on \\'python\\', ignoring\\ngalaxy.tools.deps: DEBUG: Building dependency shell command for dependency \\'fastx_toolkit\\'\\ngalaxy.tools.deps: DEBUG: Find dependency fastx_toolkit version None\\ngalaxy.tools.deps: DEBUG: Dependency fastx_toolkit not found.\\ngalaxy.tools.deps: WARNING: Failed to resolve dependency on \\'fastx_toolkit\\', ignoring\\ngalaxy.tools.deps: DEBUG: Building dependency shell command for dependency \\'cap3\\'\\ngalaxy.tools.deps: DEBUG: Find dependency cap3 version None\\ngalaxy.tools.deps: DEBUG: Dependency cap3 not found.\\ngalaxy.tools.deps: WARNING: Failed to resolve dependency on \\'cap3\\', ignoring\\ngalaxy.jobs.command_factory: INFO: Built script [/tmp/tmpHRkY17/job_working_directory/000/2/tool_script.sh] for tool command[python /home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/script_to_choose.py  trinity /tmp/tmpHRkY17/files/000/dataset_1.dat 100 100 60 > /tmp/tmpHRkY17/files/000/dataset_2.dat && echo \"done\"]\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\ngalaxy.tools.deps: DEBUG: Building dependency shell command for dependency \\'samtools\\'\\ngalaxy.tools.deps: DEBUG: Find dependency samtools version None\\ngalaxy.tools.deps: DEBUG: Dependency samtools not found.\\ngalaxy.tools.deps: WARNING: Failed to resolve dependency on \\'samtools\\', ignoring\\ngalaxy.jobs.runners: DEBUG: (2) command is: mkdir -p working; cd working; /tmp/tmpHRkY17/job_working_directory/000/2/tool_script.sh; return_code=$?; if [ -f /tmp/tmpHRkY17/job_working_directory/000/2/working/sequences_filtered.zip ] ; then cp /tmp/tmpHRkY17/job_working_directory/000/2/working/sequences_filtered.zip /tmp/tmpHRkY17/files/000/dataset_2.dat ; fi; cd \\'/tmp/tmpHRkY17/job_working_directory/000/2\\'; python \"/tmp/tmpHRkY17/job_working_directory/000/2/set_metadata__1qffo.py\" \"/tmp/tmpHRkY17/tmp/tmp95Y2R5\" \"/tmp/tmpHRkY17/job_working_directory/000/2/working/galaxy.json\" \"/tmp/tmpHRkY17/job_working_directory/000/2/metadata_in_HistoryDatasetAssociation_2_eDLhnz,/tmp/tmpHRkY17/job_working_directory/000/2/metadata_kwds_HistoryDatasetAssociation_2_Uk33zd,/tmp/tmpHRkY17/job_working_directory/000/2/metadata_out_HistoryDatasetAssociation_2_3qsAvQ,/tmp/tmpHRkY17/job_working_directory/000/2/metadata_results_HistoryDatasetAssociation_2_RwmhTW,/tmp/tmpHRkY17/files/000/dataset_2.dat,/tmp/tmpHRkY17/job_working_directory/000/2/metadata_override_HistoryDatasetAssociation_2_nCTFNi\" 5242880; sh -c \"exit $return_code\"\\ngalaxy.jobs.runners.local: DEBUG: (2) executing job script: /tmp/tmpHRkY17/job_working_directory/000/2/galaxy_2.sh\\ngalaxy.jobs: DEBUG: (2) Persisting job destination (destination id: planemo_dest)\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/jobs/5729865256bc2525?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\ngalaxy.jobs.runners.local: DEBUG: execution finished: /tmp/tmpHRkY17/job_working_directory/000/2/galaxy_2.sh\\ngalaxy.jobs.output_checker: DEBUG: Tool produced standard error failing job - [python: can\\'t open file \\'/home/umr7144/abice/vmataigne/Documents/AdaptSearch/adaptsearch-master/galaxy_wrappers/01_Filter_Assemblies/01_Choose_One_variants_per_locus_TRINITY_v1.0.py\\': [Errno 2] No such file or directory\\nTraceback (most recent call la]\\ngalaxy.jobs: DEBUG: (2) setting dataset 2 state to ERROR\\ngalaxy.jobs: DEBUG: job 2 ended (finish() executed in (509.815 ms))\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/jobs/5729865256bc2525?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a/contents?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a/contents/5729865256bc2525?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a/contents/5729865256bc2525/provenance?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/jobs/5729865256bc2525?full=true&key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/jobs/5729865256bc2525?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a/contents?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a/contents/5729865256bc2525?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTP connection (1): localhost\\nrequests.packages.urllib3.connectionpool: DEBUG: \"GET /api/histories/2891970512fa2d5a/contents/5729865256bc2525/provenance?key=9e80aa6f6b297b688782dbb8ea7f4533 HTTP/1.1\" 200 None\\n--------------------- >> end captured logging << ---------------------'", "problem_type": "functional.test_toolbox.JobOutputsError"}, "id": "functional.test_toolbox.TestForTool_filter_assemblies.test_tool_000000", "has_data": true}], "version": "0.1", "exit_code": 1, "summary": {"num_skips": 0, "num_errors": 0, "num_failures": 1, "num_tests": 1}}